src/main/java/matteroverdrive/client/render/entity/EntityRendererMutantScientist.java

```diff
@@ -39,7 +39,6 @@ public EntityRendererMutantScientist(RenderManager renderManager)
 		super(renderManager, new ModelHulkingScientist(), 0, 1);
 	}
 
-	@Override
```

Comment:
If there was an error because the method being overriden doesn't exist/have the same signature, then you should fix the method so it _does_ have the same signature. Odds are, that method still exists, it was just renamed or the signature changed. Removing the Override annotation fixes the symptom, not the cause. Fix the cause.